astroid/nodes/node_classes.py

```diff
@@ -947,7 +947,7 @@ def is_argument(self, name):
             return True
         return (
             self.find_argname(name, rec=True)[1] is not None
-            or self.kwonlyargs
+            or bool(self.kwonlyargs)
```

Comment:
Mypy prefers explicit casting.

Reply:
We were maybe returning ``self.kwonlyargs`` and not a boolean some of the time (which could be used in a context where it's going to be truthey or falsey, but not necessarily). What do you think @DanielNoord ?

Reply:
I think this is just a by product of the typing not being complete. The bool cast shouldn't be necessary so let's remove for now until we have actual complete typing for `mypy` to work with.